src/libsystemd/sd-journal/journal-verify.c

```diff
@@ -1367,8 +1386,10 @@ int journal_file_verify(
 
         if (first_contained)
                 *first_contained = le64toh(f->header->head_entry_realtime);
+#if HAVE_GCRYPT
         if (last_validated)
-                *last_validated = last_sealed_realtime;
+                *last_validated = last_tag_realtime + f->fss_interval_usec;
```

Comment:
Please use `usec_add()`.

Reply:
`last_tag_realtime` is currently not a `usec_t`, so should that be changed as well?

Reply:
OK. Let's do that in a later PR consistently. So, feel free to keep it now.